app/controllers/api/v1/classifications_controller.rb

```diff
@@ -34,6 +34,14 @@ def project
     index
   end
 
+  def offset
+    @controlled_resources = Classification.where("project_id = ? AND id > ?", params[:project_id], params[:last_id])
+    .includes(:project)
+    .includes(:user)
+    render json_api: serializer.page(params, controlled_resources, context),
```

Comment:
you should just be able to chain the where clause here. e.g. 
``` ruby
Classification
.where("project_id = ? AND id > ?", params[:project_id], params[:last_id])
.joins(:project)
.merge(Project.scope_for(:update, user)
.includes(:subjects)) 
```
if you don't include the project scope for any authenticated user will be able to use this to harvest classifications for a project_id. The Project.scope_for ensure the user has update privs on the project (collab / owner)